storage/buckets/buckets_test.go

```diff
@@ -102,6 +102,34 @@ func TestIAM(t *testing.T) {
 	if err := removeBucketIamMember(ioutil.Discard, bucketName); err != nil {
 		t.Errorf("removeBucketIamMember: %v", err)
 	}
+
+	// Uniform bucket-level access is required to use IAM with conditions.
+	if err := enableUniformBucketLevelAccess(ioutil.Discard, bucketName); err != nil {
+		t.Fatalf("enableUniformBucketLevelAccess:  %v", err)
+	}
+
+	role := "roles/storage.objectViewer"
+	member := "group:cloud-logs@google.com"
+	title := "title"
+	description := "description"
+	expression := "resource.name.startsWith(\"projects/_/buckets/bucket-name/objects/prefix-a-\")"
+	err := addBucketConditionalIamBinding(
```

Comment:
Please do this on a single line -- Go doesn't have a line length limit. You can also declare `err` inline in the if statement.

This may be an indicator there are just too many args for the sample.

https://github.com/golang/go/wiki/CodeReviewComments#line-length
https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

Reply:
Done